### DISCUSSION TITLE
Ops.Gl.Meshes.Cone cannot produce a mesh with 1 stack

### DISCUSSION THREAD

**Discussion by KOLEKTYW-online** [cables-gl/cables_docs#363] [! Welcome - readme first!]
Cone op can't produce a cone with one stack. Minimum is 2, so by default it subdivides the cone sides.

I changed line 48 from:
 if (nstacks < 2)nstacks = 2;

to:
 if (nstacks < 1)nstacks = 1;

Works fine, produces a cone without a subdivision of the sides.
<img width="384" alt="2022-10-08 02_17_12-Mandelbulb" src="https://user-images.githubusercontent.com/104379490/194677873-51a8cca2-d13d-4955-8254-a61cf295909d.png">

**Comment by pandrr** [comment]
done
